examples/response_stream/public/containers/app/pages/page_reducer_stream/index.tsx

```diff
@@ -58,7 +49,7 @@ export const PageReducerStream: FC = () => {
     RESPONSE_STREAM_API_ENDPOINT.REDUCER_STREAM,
     '1',
     { compressResponse, flushFix, simulateErrors },
-    { reducer: reducerStreamReducer, initialState }
+    { reducer: reducerStreamReducer, initialState: getInitialState() }
```

Comment:
getInitialState() will be called every render here even though it's already been used to create the reducer initially. Is that what we want?

Reply:
updated in [73d148a](https://github.com/elastic/kibana/pull/182690/commits/73d148ac05d387b36c6e25d943e58142438b013a).